src/Base/Utility.cpp

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2021-2022 The AMReX Community
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#include <AMReX_Utility.H>
+#include <pybind11/pybind11.h>
+#include <sstream>
+
+namespace py = pybind11;
+using namespace amrex;
+
+void init_Utility(py::module& m)
+{
+    m.def("concatenate",
+          [] (const std::string& root,
+                     int                num,
+                     int                mindigits) {
+          return amrex::Concatenate(root, num, mindigits);
+          }, py::return_value_policy::move,
+          "Builds plotfile name");
```

Comment:
Since `amrex::Concatenate` is very simple in terms of arguments and has no overloads, we can simply use it's pointer.

Then, we can use the default return policy, copying the string out to Python by value.

Lastly, let's name the user-facing function arguments with `py::arg`:
```suggestion
          &amrex::Concatenate,
          "Builds plotfile name",
          py::arg("root"), py::arg("num"), py::arg("mindigits")=5
    );
```